lib/locasms.rb

```diff
@@ -6,9 +6,11 @@
 autoload :Logger,     'logger'
 
 module LocaSMS
-  autoload :Client,     'locasms/client'
-  autoload :Exception,  'locasms/exception'
-  autoload :Numbers,    'locasms/numbers'
+  autoload :Client, 'locasms/client'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.